examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -256,28 +257,36 @@ void sl_wfx_host_si91x_sleep_wakeup()
  * @brief
  *       Setting the RS911x in DTIM sleep based mode
  *
- * @param[in] None
+ * @param[in] sl_si91x_ble_state : State to set for the BLE
+              sl_si91x_wifi_state : State to set for the WiFi
  * @return
  *        None
  *********************************************************************/
-int32_t wfx_rsi_power_save()
+int32_t wfx_rsi_power_save(rsi_power_save_profile_mode_t sl_si91x_ble_state, sl_si91x_performance_profile_t sl_si91x_wifi_state)
 {
     int32_t status;
-    status = rsi_bt_power_save_profile(2, 0);
+
+    status = rsi_bt_power_save_profile(sl_si91x_ble_state, 0);
     if (status != RSI_SUCCESS)
     {
         SILABS_LOG("BT Powersave Config Failed, Error Code : 0x%lX", status);
         return status;
     }
-
-    sl_wifi_performance_profile_t wifi_profile = { .profile = ASSOCIATED_POWER_SAVE };
+    sl_wifi_performance_profile_t wifi_profile = { .profile = sl_si91x_wifi_state };
```

Comment:
how many other members does this structure have? Since we are only setting the value of .`profile` other will be left with an undefined value. Can this be problematic?

Reply:
Based on C standard (6.7.9 )this is ok and the rest will be implicitly init 

> 21. If there are fewer initializers in a brace-enclosed list than there are elements or members
> of an aggregate, or fewer characters in a string literal used to initialize an array of known
> size than there are elements in the array, the remainder of the aggregate shall be
> initialized implicitly the same as objects that have static storage duration.